pkg/operator/ceph/csi/version.go

```diff
@@ -25,16 +25,14 @@ import (
 )
 
 var (
-	//minimum supported version is 3.5.0
-	minimum = CephCSIVersion{3, 5, 0}
+	//minimum supported version is 3.7.0
+	minimum = CephCSIVersion{3, 7, 0}
```

Comment:
Maybe merged too soon... Setting the min version to 3.7 will make upgrade difficult. If someone is using any different version, this means they won't be able to upgrade to rook v1.10 unless they also update their csi image at the same time. If we set the min version to 3.6, it will give some time buffer to allow them to change the csi driver later. What is the requirement now to set the min version to 3.7? If this is really a requirement, we will just need to be specific in the upgrade guide about the strict requirement.

Reply:
@humblec @Madhu-1 ^

Reply:
Thanks, @travisn , There is no hard requirement to keep 3.7.0 as default changing it back to 3.6.0 here https://github.com/rook/rook/pull/10774

Reply:
ack.. its good to keep the minimum version on 3.6.0